app/serializers/skill.js

```diff
@@ -0,0 +1,4 @@
+import DS from 'ember-data';
+
+export default DS.RESTSerializer.extend({
```

Comment:
It is actually now:

export default DS.RESTAdapter.extend({
  namespace: 'api/v1'
});

so I think it's best that it's multi-line.